src/lib/mina_networking/rpcs.ml

```diff
@@ -831,6 +831,33 @@ module Get_completed_snarks = struct
     include Master
   end)
 
+  module V1 = struct
+    module T = struct
+      type query = unit [@@deriving sexp]
+
+      type response = Transaction_snark_work.Checked.Stable.V2.t list option
```

Comment:
Change this to unchecked? Not sure why it changed back to checked

Reply:
i made a new unchecked versioned type to be used here instead

Reply:
done

Reply:
Hmm, I think this slipped in the previous PR. Even https://github.com/MinaProtocol/mina/blob/ab2e50bbffbc157993b7a597616b21e2e355dc4f/src/lib/mina_networking/rpcs.ml#L807 needs to be the unchecked version or simply `Transaction_snark_work.t`

Reply:
it did, i need this to be a versioned unchecked type in the rpc response